app/views/generic_files/upload/_to_collection.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>
+  <label><%= t("sufia.upload.collection.select_label") %></label>
+  <%= select_tag 'collection', options_from_collection_for_select(@user_collections, "id", "title") %>
```

Comment:
This should probably be `collection_id` as it is the Rails convention to append `_id` when you're talking about the identifier that underlies the relationship.